docs/tutorials/mlflow.ipynb

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
<p>Is there no configuration needed for MLFlow at all? What if I'm already using MLFlow with any given configuration, will it work for me OOTB?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
yes it should work